### PR TITLE
br-stream: add precheck before create task

### PR DESCRIPTION
**Pull request by 3pointer**
### What problem does this PR solve?

Issue Number: close https://github.com/pingcap/tidb/issues/33087

Problem Summary:
make sure that all stores support log backup then create log backup tasks.

### What is changed and how it works?
1. add a check before task starts.
2. use tikv status_address to get config and check when log backup is enabled.

### Check List

Tests 

- [x] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

### Release note

```release-note
None
```

### CODE REVIEW

**Comment by joccau** [comment]
rest LGTM

**Comment by joccau** [comment]
(https://ci.pingcap.net/blue/organizations/jenkins/tidb_ghpr_check/detail/tidb_ghpr_check/54908/pipeline#step-73-log-213)[2022-03-15T09:58:01.697Z] br/pkg/task/stream.go:22: File is not `goimports`-ed (goimports)

[2022-03-15T09:58:01.697Z] 	"github.com/pingcap/kvproto/pkg/metapb"

[2022-03-15T09:58:01.697Z] 	"github.com/pingcap/tidb/br/pkg/httputil"

[2022-03-15T09:58:01.697Z] br/pkg/task/stream_test.go:19: File is not `goimports`-ed (goimports)

[2022-03-15T09:58:01.697Z] 	"fmt"

[2022-03-15T09:58:01.697Z] br/pkg/task/stream.go:359:28: response body must be closed (bodyclose)

[2022-03-15T09:58:01.697Z] 			resp, e := s.httpCli.Get(addr)

[2022-03-15T09:58:01.697Z] 			                        ^

[2022-03-15T09:58:01.697Z] level=info msg="File cache stats: 1702 entries of total size 26.3MiB"

[2022-03-15T09:58:01.697Z] level=info msg="Memory: 2299 samples, avg is 1943.3MB, max is 10864.1MB"

[2022-03-15T09:58:01.697Z] level=info msg="Execution took 4m7.969458773s"

[2022-03-15T09:58:01.697Z] make: *** [check-static] Error 1

File: br/pkg/utils/safe_point.go [hunk 1]
```diff
@@ -137,3 +138,12 @@ func StartServiceSafePointKeeper(
 	}()
 	return nil
 }
+
+type FakePDClient struct {
+	pd.Client
+	Stores []*metapb.Store
+}
+
+func (c FakePDClient) GetAllStores(context.Context, ...pd.GetStoreOption) ([]*metapb.Store, error) {
```
[the remark this answers is not in the document]
Comment (3pointer) [reply br/pkg/utils/safe_point.go:148]: done

**Review by YuJuncen — APPROVED** [review]
LGTM

**Review by joccau — APPROVED** [review]
LGTM

**Pull request merged**